### PR TITLE
Add shft (shft.co)

### PR DESCRIPTION
**Pull request by hcyildirim**
Thanks for bringing this project to life!

### CODE REVIEW

File: src/_companies/europe/turkiye/istanbul/shft/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+stack:
+  - rails
+  - postgres
+  - react
+  - mysql
+
+created_at: 2023-03-28 10:38:16
+---
+
+SHFT is a startup for the fast-growing tech companies. We take the development load off your shoulders with scalable and dedicated in-house developers. Think of us as your strategic product partner that codes. We welcome the enthusiastic startups and target-driven scale-ups; assist them with a thorough design, accompanied by a detailed MVP scope, followed by an impeccable development, and finally a confident product launch. We essentially cover your whole product building phase.
```
Comment (CalvinWalzel) [inline src/_companies/europe/turkiye/istanbul/shft/index.md:26]: Could you change this to be in the third person? It's the default for all other company descriptions :)

**Review by CalvinWalzel — COMMENTED** [review]
Hey @hcyildirim, thanks for your submission! I've added one comment with feedback. Could you let me know what you think?

File: src/_companies/europe/turkiye/istanbul/shft/index.md (hunk 1 above)
Comment (hcyildirim) [reply src/_companies/europe/turkiye/istanbul/shft/index.md]: I didn't realize it was that way. Changed it, thank you!

**Review by CalvinWalzel — APPROVED** [review]
Thanks for the submission :)

**Pull request merged**